aldryn_newsblog/models.py

```diff
@@ -235,7 +235,7 @@ def save(self, *args, **kwargs):
 
         # Ensure we aren't colliding with an existing slug *for this language*.
         if not Article.objects.translated(
-                slug=self.slug).exclude(pk=self.pk).exists():
+                slug=force_unicode(self.slug)).exclude(pk=self.pk).exists():
```

Comment:
Oops, you're right about force_unicode(), which is actually already force_text() in the right envs.

Still not crazy about allowing the force_unicode() here rather than putting it in your implementation where the problem is caused. You'd be better served by putting the coercion there, but, in the spirit of defensiveness :+1: